src/org/zalando/stups/essentials/api.clj

```diff
@@ -19,13 +19,14 @@
             [org.zalando.stups.friboo.user :as u]
             [org.zalando.stups.friboo.config :refer [require-config]]
             [org.zalando.stups.essentials.sql :as sql]
+            [org.zalando.stups.essentials.external.kio :as kio]
             [io.sarnowski.swagger1st.util.api :refer [throw-error]]
             [ring.util.response :refer :all]
             [clojure.string :as str]
             [clojure.set :as set]))
 
 ; define the API component and its dependencies
-(def-http-component API "api/essentials-api.yaml" [db])
+(def-http-component API "api/essentials-api.yaml" [db tokens])
```

Comment:
What about using the incoming token (part of the request parameter) to perform the subsequent call? This way "essentials" micro service would not have to maintain its own service user tokens.